sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -25,6 +25,16 @@ public class RecordedTestSanitizer
 
         private static readonly string[] s_sanitizeValueArray = { SanitizeValue };
 
+        public RecordedTestSanitizer()
+        {
+            // Lazy sanitize fields in the request and response bodies
+            AddJsonPathSanitizer("$..primaryKey");
```

Comment:
It may be worth updating the readme to call out the default path sanitizers, or at least link to the place in code where they are specified - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#sanitizing